AOC_Day1/ContentView.swift

```diff
@@ -14,53 +14,43 @@ struct ContentView: View {
     
     
     func rollingTotals(readings: [Int]) -> [Int] {
-        var returnArray: [Int] = []
+        var rollingTotals: [Int] = []
         for i in 0..<readings.count {
-            if i < 2 {
-                continue
-            } else {
-                returnArray.append( readings[i-2] + readings[i-1] + readings[i] )
+            if i >= 2 {
+                rollingTotals.append( readings[i-2] + readings[i-1] + readings[i] )
             }
         }
-        return returnArray
+        return rollingTotals
     }
     
     
     func countIncreases(readings: [Int]) -> Int {
         var counter = 0
         for i in 0..<readings.count {
-            if i == 0 {
-                continue
-            } else {
-                if readings[i] > readings[i-1] {
-                    counter += 1
-                }
+            if i > 0, 
+                readings[i] > readings[i-1] {
+                counter += 1
             }
         }
         return counter
     }
     
     
-    func loadFile(fileName: String) -> [Int] {
-        if let filepath = Bundle.main.path(forResource: fileName, ofType: "txt") {
-            do {
-                var returnArray: [Int] = []
-                
-                let contents = try String(contentsOfFile: filepath)
-                let stringArray = contents.components(separatedBy: "\n")
-                
-                for i in 0..<stringArray.count - 1 {
-                    returnArray.append(Int(stringArray[i]) ?? 0 )
-                }
-                return returnArray
-                
-            } catch {
-                // contents could not be loaded
+    func loadFile(fileName: String) -> [Int]? {
+        guard let filepath = Bundle.main.path(forResource: fileName, ofType: "txt"),
+              let contents = try String(contentsOfFile: filepath) else {
+            return nil
+        }
+        var returnArray: [Int] = []
+
+        let stringArray = contents.components(separatedBy: "\n")
+
+        for i in 0..<stringArray.count - 1 {
+            if let intValue = Int(stringArray[i]) {
```

Comment:
`if let` is great for unwrapping optional, i.e. here so you don't have to use the nil coalesce to `0`.
In this case, we now only add valid values: I suspect this means you can also use `stringArray.count` rather than needing `-1`.

This would also be a great place to consider using `compactMap` instead which will transform your array:
`let returnArray = stringArray.compactMap(Int.init)` would concisely transform your string array.